### PR TITLE
Various css fixes

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: src/views/about.njk [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 {% block content %}
 <main>
+  <header class="main-header">
+    <div class="main-title">
+      <h1>About The Federation</h1>
```
Comment (jaywink) [inline src/views/about.njk]: Hmm to my ear this kinda doesn't sit.. what about just `The Federation`?

File: src/views/nodes-list.njk [hunk 2]
```diff
@@ -2,13 +2,20 @@
 
 {% block content %}
 <main>
+  <header class="main-header">
+    <div class="main-title">
+      <h1>All the nodes</h1>
```
Comment (jaywink) [inline src/views/nodes-list.njk]: Again, doesn't quite sit in my English brain :) Maybe `Nodes`, `Nodes list` or `List of all nodes`?

File: src/views/about.njk (hunk 1 above)
Comment (Flaburgan) [inline src/views/about.njk]: The federation is the title of the home page. This is for the "About" page. But yeah I'm open to something better.

Comment (jaywink) [inline src/views/about.njk]: Well I don't see why it can't have a simple title :) Just `About` then?

File: src/views/nodes-list.njk (hunk 2 above)
Comment (jaywink) [inline src/views/nodes-list.njk]: `Nodes`?

Comment (jaywink) [inline src/views/nodes-list.njk]: "All the nodes" is a bit meme ;)

**Comment by Flaburgan** [comment]
Texts updated.

**Comment by Flaburgan** [comment]
The button matching the chart is now displayed in blue, I removed the unused vendors (see #96).

File: src/views/about.njk [hunk 3]
```diff
@@ -4,7 +4,8 @@
 <main>
   <header class="main-header">
     <div class="main-title">
-      <h1>About The Federation</h1>
+      <h1>About</h1>
+      <h2>Behind the scene</h2>
```
Comment (jaywink) [inline src/views/about.njk]: "Behind the scenes" would be more correct - sorry, have to point these small grammar things out ;)

Comment (Flaburgan) [inline src/views/about.njk]: Thanks I need that!

**Comment by Flaburgan** [comment]
>Hmmm you're removing also the rickshaw css files - they're needed for the graphs are they not?

I'm only loading the rickshaw.min.css as far as I can see so it should be ok.

**Comment by jaywink** [comment]
Thanks man! <3

**Pull request merged**